include/ale/Orientations.h

```diff
@@ -71,15 +71,27 @@ namespace ale {
       bool invert=false
     ) const;
 
+    /**
+     * Add an additional constant multiplication.
+     * This is equivalent to left multiplication by a constant rotation
+     */
+    Orientations &addConstantRotation(const Rotation &addedConst);
+
     /**
      * Multiply this orientation by another orientation
      */
     Orientations &operator*=(const Orientations &rhs);
+
     /**
      * Multiply this orientation by a constant rotation
      */
     Orientations &operator*=(const Rotation &rhs);
 
+    /**
+     * Invert the orientations
```

Comment:
Note for myself that I need to note all of the stuff that doesn't perfectly invert in here.